lib/rubocop/cop/rails/link_to_blank.rb

```diff
@@ -35,6 +35,7 @@ class LinkToBlank < Cop
           (pair {(sym :rel) (str "rel")} (str _))
         PATTERN
 
+        # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Added this because the rubocop started triggering on:
```
lib/rubocop/cop/rails/link_to_blank.rb:38:9: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for on_send is too high. [8/7]
        def on_send(node) ...
        ^^^^^^^^^^^^^^^^^
```

Reply:
Can you open this change as a separate PR?

Reply:
added #266